plugin/clone/include/clone_client.h

```diff
@@ -51,6 +51,8 @@ using Time_Msec = std::chrono::milliseconds;
 using Time_Sec = std::chrono::seconds;
 using Time_Min = std::chrono::minutes;
 
+extern const char *clone_compression_algorithms[3];
```

Comment:
This (and the corresponding declaration in `clone_client.cc`) are not specific to the client, but general to both sides. IMHO should go to `clone_plugin.h`/`.cc`

Reply:
Will this introduce circular dependency between clone_plugin.h and clone_client.h? The clone_plugin.h already include clone_client.h. With this change, the clone_client will need to include clone_plugin.h as well.